### PR TITLE
V6/fix ens > manage domain

### PR DESCRIPTION
**Pull request by hsurf22**
### Bug

* \[x] Updated CHANGELOG.md
* \[x] Add PR label
   - Fix ens > manage domain layout

### CODE REVIEW

**Review by gamalielhere — APPROVED** [review]

**Review by gesseekur — APPROVED** [review]

File: src/dapps/ens-manager-dapp/TheENSManagerLayout.vue [hunk 1]
```diff
@@ -61,7 +61,7 @@
     =====================================================================================
     -->
       <template #tabContent2>
-        <v-sheet min-height="500px" class="pa-12">
+        <v-sheet min-height="500px" class="pa-4 pa-md-12">
```
Comment (olgakup) [inline src/dapps/ens-manager-dapp/TheENSManagerLayout.vue:64]: can you double check pa-4? i think we are using pa-3 on xs screens

File: src/dapps/ens-manager-dapp/TheENSManagerLayout.vue [hunk 2]
```diff
@@ -108,8 +107,11 @@
                     >
                       <v-icon small class="call-made"> mdi-call-made </v-icon>
                     </a>
-                  </div>
-                  <div class="d-flex align-center justify-end">
+                  </v-col>
+
+                  <v-spacer></v-spacer>
+
+                  <v-col cols="12" lg="6" class="d-flex align-center">
```
Comment (olgakup) [inline src/dapps/ens-manager-dapp/TheENSManagerLayout.vue:114]: should it be md="6"m instead of lg="6"?

**Review by olgakup — CHANGES_REQUESTED** [review]

File: src/dapps/ens-manager-dapp/TheENSManagerLayout.vue (hunk 1 above)
Comment (hsurf22) [reply src/dapps/ens-manager-dapp/TheENSManagerLayout.vue:64]: fixed

File: src/dapps/ens-manager-dapp/TheENSManagerLayout.vue (hunk 2 above)
Comment (hsurf22) [reply src/dapps/ens-manager-dapp/TheENSManagerLayout.vue:114]: fixed

**Pull request merged**